### PR TITLE
chore(ImmutableDevices): add extra props to immutable table

### PR DESCRIPTION
**Pull request by mkholjuraev**
This adds extra props to the ImmutableDevices module to make it more usable. Cypress tests are added.

### CODE REVIEW

File: src/components/ImmutableDevices/ImmutableDevices.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ const ImmutableDevices = ({
   filterConfig,
   hideFilters,
   activeFiltersConfig,
+  tableActions,
+  onRefresh,
+  actionsConfig,
```
Comment (bastilian) [inline src/components/ImmutableDevices/ImmutableDevices.js:21]: Why not add a `...props` and use it below with the `InventoryTable`?

```js
<InventoryTable
   showTags
   ...
   { ...props }
/>
```

Later on if we see that certain attributes are used all over the place with the ImmutableDevices component we can add them here with proper default.

Comment (mkholjuraev) [reply src/components/ImmutableDevices/ImmutableDevices.js:21]: Done.

**Review by bastilian — APPROVED** [review]
Approved! Thank you @mkholjuraev!

**Pull request merged**